.gitattributes

```diff
@@ -5,6 +5,7 @@
 /docs               export-ignore
 /tests              export-ignore
 /lib/Imagine/Test   export-ignore
+/resources/Adobe/Profile Information.pdf export-ignore
```

Comment:
path is `/lib/Imagine/resources/Adobe/Profile`

Reply:
Done (one year ago)